core/value.asdl

```diff
@@ -27,6 +27,8 @@ module value
 
   IntBox = (int i)
 
+  InitializerValue = (str? key, str rval, bool plus_eq)
```

Comment:
I think it might be cleaner to use frontend/syntax.asdl assign_op everywhere, although we can also refactor that separately probably